docs/plugins.md

```diff
@@ -536,6 +536,41 @@ Enable it manually on each run via `-p` option:
 
     npx codeceptjs run -p pauseOnFail
 
+## playwrightCoverage
+
+Dumps playwright code coverage after every test.
+
+#### Configuration
+
+Configuration can either be taken from a corresponding helper (deprecated) or a from plugin config (recommended).
+
+```js
+plugins: {
+   playwrightCoverage: {
+     enabled: true
+   }
+}
+```
+
+Possible config options:
+
+-   `coverageDir`: directory to dump coverage files
+-   `uniqueFileName`: generate a unique filename by adding uuid
+
+    First of all, your mileage may vary!
+
+    To work, you need the client javascript code to be NOT uglified. They need to be built in "development" mode.
+    And the end of your tests, you'll get a directory full of coverage per test run.  Now what?
+    You'll need to convert the coverage code to something istanbul can read.
```

Comment:
Would it makes sense to add the Conversion to Istanbul as well?

Reply:
I am inclined that might be a fair suggestion. As, if we do so `codecept` will be exposing a single unified `istanbul` format, which I believe is widely accepted and then users can covert appropriately.

@DavertMik thoughts on this as the original implementation didn't do the same, I want to clarify if there was any reason for the same or not?